### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #47 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.
- Update to`marked` & `prism` src used in demo index.html so that it works for latest `marked` npm package.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by fajar-apri-alaska** [comment]
Hi @jason-capsule42 or @blackfalcon.
I'm not sure how to fix this `npm ci` error for `node 11` [in this pipeline error.](https://github.com/AlaskaAirlines/auro-card/actions/runs/6796061012/job/18475370018?pr=48#step:4:97)

Please advise.

FYI some references:
https://stackoverflow.com/questions/9768444/possible-eventemitter-memory-leak-detected

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
Tests are failing here. Please check whythat is.

File: demo/index.html [hunk 1]
```diff
@@ -10,13 +10,16 @@
...
     <main></main>
 
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/prismjs/prism.js"></script>
+
     <script type="module">
-      import "https://unpkg.com/marked@latest/marked.min.js";
-      import "https://unpkg.com/prismjs@latest/prism.js";
+      // import "https://unpkg.com/marked@latest/marked.min.js";
+      // import "https://unpkg.com/prismjs@latest/prism.js";
```
Comment (blackfalcon) [inline demo/index.html:18]: Please see https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/demo/index.html for the correct configuration. 

Also, please remove commented code when prepared for PR review.

File: package.json [hunk 2]
```diff
@@ -35,6 +35,8 @@
     "@alaskaairux/design-tokens": "^3.1.0",
```
Comment (blackfalcon) [inline package.json:35]: Remove

File: src/style-banner.scss [hunk 3]
```diff
@@ -4,11 +4,14 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables";
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties";
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties";
```
Comment (blackfalcon) [inline src/style-banner.scss:8]:
```suggestion
```

File: src/style-banner.scss [hunk 4]
```diff
@@ -4,11 +4,14 @@
...
 
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints";
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/core";
+@import "./../node_modules/@aurodesignsystem/webcorestylesheets/src/breakpoints";
+@import "./../node_modules/@aurodesignsystem/webcorestylesheets/src/core";
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
Comment (blackfalcon) [inline src/style-banner.scss:14]:
```suggestion
```

File: src/style.scss [hunk 5]
```diff
@@ -4,9 +4,12 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties";
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties";
```
Comment (blackfalcon) [inline src/style.scss:7]:
```suggestion
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by fajar-apri-alaska** [comment]
I have pushed some more changes per feedback above.

**Review by jordanjones243 — APPROVED** [review]

**Review by jordanjones243 — CHANGES_REQUESTED** [review]
Tests are failing. Support for Node 11 needs to be removed from the repo.

**Comment by jordanjones243** [comment]
Closing this PR as the work was handled in a different PR:
- https://github.com/AlaskaAirlines/auro-card/pull/52

**Pull request closed without merging**